inc/environment_indicator/namespace.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Altis\Cloud\Environment_Indicator;
+
+use function Altis\get_environment_name;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'admin_bar_init', __NAMESPACE__ . '\\enqueue_admin_scripts' );
+	add_action( 'admin_bar_menu', __NAMESPACE__ . '\\add_admin_bar_env_info' );
+}
+
+/**
+ * Add menu item to admin bar to display environment info:
+ * - environment type
+ * - environment name
+ * - URL to Altis dashboard for the environment
+ */
+function add_admin_bar_env_info() {
+	if ( ! is_admin_bar_showing() ) {
+		return;
+	}
+
+	// Specify capabilities for which environment indicator is shown in the admin bar.
+	$capability = apply_filters( 'altis_env_indicator_capability', 'manage_options' );
+	if ( ! current_user_can( $capability ) ) {
+		return;
+	}
+
+	global $wp_admin_bar;
+	// Add environment menu item to the admin bar.
+	$wp_admin_bar->add_menu( [
+		'id'   => 'altis-env-indicator',
+		'meta' => [
+			'class' => 'altis-env-indicator--' . get_environment_type(),
+		],
+	] );
+
+	// Environment type sub-menu item.
+	$env_type_text = strtoupper( get_environment_type() );
+	$env_type_text = apply_filters( 'altis_env_indicator_text', $env_type_text, get_environment_type() );
```

Comment:
I'm not sure this filter is really necessary. I think instead we could have an array of translatable strings eg:
```php
$envs = [
    'local' => _x( 'local', 'Server environment type', 'altis' ),
    'development' => _x( 'dev', 'Server environment type', 'altis' ),
    'staging' => _x( 'staging', 'Server environment type', 'altis' ),
    'production' => _x( 'prod', 'Server environment type', 'altis' ),
];
```